boards/stm32g031-disco/doc.txt

```diff
@@ -0,0 +1,30 @@
+/**
+ * @defgroup    boards_stm32g031-disco STM32G031-DISCO
+ * @ingroup     boards
+ * @brief       Support for the STM32G031-DISCO board.
+ *
+ * ### General information
+ *
+ * The ST [STM32G031-DISCO](https://www.st.com/en/evaluation-tools/stm32g0316-disco.html)
+ * is an evaluation board supporting a ARM Cortex-M7 STM32F723IE microcontroller
```

Comment:
That's a beefier CPU than I was expecting on this board :wink:

Reply:
bah! fixed, thanks. :stuck_out_tongue_closed_eyes: